components/setting_item_max.tsx

```diff
@@ -1,131 +1,53 @@
 // Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
 // See LICENSE.txt for license information.
 
-import PropTypes from 'prop-types';
-import React from 'react';
+import React, {ReactNode} from 'react';
 import {FormattedMessage} from 'react-intl';
 
 import SaveButton from 'components/save_button';
 import Constants from 'utils/constants';
 import {isKeyPressed} from 'utils/utils';
 
-export default class SettingItemMax extends React.PureComponent {
+export type Props = {
+    inputs: ReactNode;
+    containerStyle?: string;
+    clientError?: string | React.ReactNode | null;
```

Comment:
`ReactNode` is already typed with `string` and `null` (in addition to some other types), so you can just replace this with a single `React.ReactNode`